src/worker/generate-kmp.ts

```diff
@@ -24,9 +24,7 @@ export async function generateKmp(
 ): Promise<ArrayBuffer> {
   const kmpJsonFile = generateKmpJson({
     languages: [{ name: langName, id: bcp47Tag }],
-    // TODO: change generateKmpJson() to accept a modelID argument instead of splitting it here:
-    authorID: modelId.split(".")[0],
-    modelTag: modelId.split(".")[2],
+    modelId: modelId,
```

Comment:
```suggestion
    modelID: modelId,
```

The existing variable, called `modelId` can be changed in a different pull request!